int8 longitudinal_value
int8 vertical_value
int8 rotational_value
int8 gain_value
